jekyll/_cci2/deploy-to-amazon-sagemaker.adoc

```diff
@@ -0,0 +1,364 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+---
+= Deploy to Amazon SageMaker
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A short page description goes here
+:icons: font
+:experimental:
+
+NOTE: CircleCI releases is currently in **open preview**. During open preview release features are available for free. Charges will be associated with release features once they are generally available. The link:https://circleci.com/developer/orbs/orb/circleci/aws-sagemaker[SageMaker orb] is also in **open preview**.
+
+This how-to guide walks you through the steps to set up and use the Amazon SageMaker orb to deploy and release models to SageMaker. You will track releases in the CircleCI release dashboard. This gives you the ability to manage promotions and rollbacks. You will orchestrate model deployments to endpoints across different environments. The link:https://github.com/CircleCI-Public/sagemaker-deploy-examples#[example project repository] allows you to train a new model package version and then deploy it across multiple environments.
+
+For further information about the orb and the releases feature see the following:
+
+* xref:/release/releases-overview#[Releases overview]
+* link:https://circleci.com/developer/orbs/orb/circleci/aws-sagemaker[Amazon SageMaker orb]
+
+[#prerequisites]
+== Prerequisites
+
+To follow along with this how-to guide you will need the following:
+
+* A CircleCI account connected to your code. You can link:https://circleci.com/signup/[sign up for free]
+* Basic knowledge of Amazon SageMaker
+* An AWS account with access to SageMaker and a SageMaker studio domain
```

Comment:
@meeech am I right that SageMaker studio is needed only if they want to follow the model training in the sample repo? 
Would it make sense to change this to
> An AWS account with access to SageMaker (and a SageMaker studio domain, if you don't already have a model and want to train one)

Reply:
@BeFunes hmmm yes. I mean, they need a model we can use to deploy, so they need a studio / w a model package registry. 

Is there a way to reword to that, and that we have a way to make the model if they aren't bringing one with them?

Reply:
We can also drop the differentiation. 
I thought they could potentially even load the model from other sources (non studio) if they wished. 
We can keep what you had originally  or separate them, but meaning remains the same

>- An AWS account with access to SageMaker
>- A SageMaker studio domain to follow along the training step, or an existing Model Package Registry if you have already trained one